File: python/funciones.py
def factorial(n):
    if n < 0:
        return "No existe factorial para negativos"

    resultado = 1

    for i in range(1, n + 1):
        resultado *= i

    return resultado

print(factorial(5))
print(factorial(-3))


def es_primo(numero):
    if numero <= 1:
        return False

    for i in range(2, numero):
        if numero % i == 0:
            return False

    return True

print(es_primo(6))

File: js/funciones.js
function factorial(n) {
    if (n < 0) {
        return "No existe factorial para negativos";
    }

    let resultado = 1;

    for (let i = 1; i <= n; i++) {
        resultado *= i;
    }

    return resultado;
}

console.log(factorial(5));
console.log(factorial(-3));


function esPrimo(numero) {
    if (numero <= 1) {
        return false;
    }

    for (let i = 2; i < numero; i++) {
        if (numero % i === 0) {
            return false;
        }
    }

    return true;
}
console.log(esPrimo(7));